core.impl/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/impl/repair/scheduler/TestScheduleManager.java

```diff
@@ -192,6 +231,57 @@ public void testGetCurrentJobStatusNoRunning() throws InterruptedException
         assertThat(myScheduler.getCurrentJobStatus()).isNotEqualTo("Job ID: " + jobId.toString() + ", Status: Running");
         latch.countDown();
     }
+
+    @Test
+    public void testRunningOneJobWithThrowingLock() throws LockException
+    {
+        DummyJob job = new DummyJob(ScheduledJob.Priority.LOW);
+        myScheduler.schedule(nodeID1, job);
+
+        when(myLockFactory.tryLock(any(), anyString(), anyInt(), anyMap())).thenThrow(new LockException(""));
+
+        myScheduler.run(nodeID1);
+
+        assertThat(job.hasRun()).isFalse();
+        assertThat(myScheduler.getQueueSize(nodeID1)).isEqualTo(1);
+    }
+
+    @Test
+    public void testTwoJobsThrowingLock() throws LockException
+    {
+        DummyJob job = new DummyJob(ScheduledJob.Priority.LOW);
```

Comment:
When having more than one jobs like this, name them all with a number, eg job1, job2 etc. Makes the similarities stand out better.